### DISCUSSION TITLE
Module '"xstate"' declares x locally, but it is not exported

### DISCUSSION THREAD

**Discussion by tukusejssirs** [statelyai/xstate#3020] [Q&A]
I tried to import some stuff from `xstate` (e.g. `import {assign, createMachine, spawn, StateNode} from 'xstate'`) in TypeScript, but it fails to import anything with `Module '"xstate"' declares x locally, but it is not exported.`

This happens on `xstate@4.29.0` and `typescript@4.5.5`.

**Comment by davidkpiano** [comment]
Can you please make a reproducible example of this? https://codesandbox.io/s/xstate-typescript-template-s9kz8?file=/src/index.ts

**Reply by tukusejssirs** [reply]
Well, the issue was elsewhere, as it runs without issue in Nest.js (it failed to run in a plain TS file). Sorry for the noise.
